endpoints.yml

```diff
@@ -0,0 +1,42 @@
+# This file contains the different endpoints your bot can use.
+
+# Server where the models are pulled from.
+# https://rasa.com/docs/rasa/model-storage#fetching-models-from-a-server
+
+#models:
+#  url: http://my-server.com/models/default_core@latest
+#  wait_time_between_pulls:  10   # [optional](default: 100)
```

Comment:
are these models used? what's my-server.com. You can just delete these lines if not needed.